File: code-examples/cpp/collation/findOneAndDelete-no-collation.cpp
/* 
   Delete the first document that contains a value greater
   than 100 in the "a" field when ordered by using the default
   binary collation order.
*/
// start findOneAndDelete no collation

bsoncxx::stdx::optional<bsoncxx::document::value> maybe_result =
    myColl.find_one_and_delete(document{} << "a" << open_document <<
                                  "$gt" << "100" << close_document << finalize);

// end findOneAndDelete no collation

File: code-examples/php/collation/findOneAndDelete-no-collation.php
<?php
/* 
   Delete the first document that contains a value greater
   than 100 in the "a" field when ordered by using the default
   binary collation order.
*/

// Create a MongoDB client
$client = new MongoDB\Client(); 

// Select a database
$db = $client->test; 

// Select a collection
$collection = $db->myColl;

// Find the first document where "a" > 100 and delete it
$result = $collection->findOneAndDelete(['a' => ['$gt'=>100]]); 

?>
